docs/infrastructure/administration.md

```diff
@@ -21,13 +23,22 @@ Production deploys are a manual process that is performed by a Treeherder admin
 !!! note
     To access treeherder-prod in [Jenkins](https://ops-master.jenkinsv2.prod.mozaws.net/job/gcp-pipelines/job/treeherder/job/treeherder-production/), cloudOps need to grant you access and you'll need to follow [these steps](https://github.com/mozilla-services/cloudops-deployment/#accessing-jenkins) to set it up before the url will work.
 
+### Using Prototype
+
+The `prototype` branch is useful and recommended for testing changes that might impact users - such as schema changes, major rewrites or very large prs, and for modifications to cron jobs or to the data ingestion pipeline. However, it's important to note that any schema changes will need to be reset after testing which might involve having cloudOps manually deleting tables or columns, and potentially modifying the django_migrations table so that it matches the current state of django migration files (in fact, this applies to all deployments).
+
+Access to push to the prototype branch requires special permission and an admin can gratn access in the repository branch settings.
+
+!!! note
+    Ensure all tests pass on the CI as they can block deployment to the prototype instance if any have failed.
```

Comment:
jbuck clarified that this is not the case, it will be deployed regardless of whether tests pass or not, so I will update this.